topics/migration.md

```diff
@@ -18,15 +18,19 @@ Refer to [install Valkey](installation.md) for installation options.
 
 ### Migration compatibility matrix
 
-You can migrate Redis server to Valkey. 
-The following table provides migration options depending on the Redis version you run:
+You can migrate a Redis server to Valkey.
+Valkey is backward compatible with Redis OSS 7.2 and all earlier open source Redis versions.
+Migrating from any open source Redis version to Valkey is just an upgrade.
+Redis 7.4 and later are not open source and the data files are no longer compatible with Valkey.
+It may be possible to migrate the data to Valkey from proprietary Redis versions and other Redis-like software but it requires another method and is not covered by this document.
 
-| Redis | Valkey |
-|-------|--------|
-| 6.x   | 7.2.x  |
-| 7.2.x | 7.2.x  |
-| 7.2.x | 8.0.x  |
-| 7.4   | n/a    |
+The following table provides migration options depending on the open source Redis version you run:
+
+| Redis                 | Valkey |
+|-----------------------|--------|
+| 2.x - 7.2.x           | 7.2.x  |
+| 2.x - 7.2.x           | 8.0    |
+| 7.4 (not open source) | n/a    |
```

Comment:
```suggestion
The following table provides migration options depending on the Redis version you run:

| Redis                 | Valkey |
|-----------------------|--------|
| OSS 2.x - 7.2.x       | 7.2.x  |
| OSS 2.x - 7.2.x       | 8.0    |
| CE 7.4                | n/a    |
```

I found it odd that you added open source in the header, but then later called it non-OSS. Maybe this is clearer?